flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -304,4 +368,60 @@ public String getDescription() {
 			return "TestOptimizerPlan <input-file-path> <output-file-path>";
 		}
 	}
+
+	public static final class TestExecuteTwice {
+
+		public static void main(String args[]) throws Exception {
```

Comment:
I was concerned about this too. Perhaps a better idea would be have these classes in the test-jar?